core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -90,6 +92,54 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  // Blacklist confs
+  private[spark] val BLACKLIST_ENABLED =
+    ConfigBuilder("spark.blacklist.enabled")
+      .booleanConf
+      .createOptional
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_EXECUTOR =
+    ConfigBuilder("spark.blacklist.task.maxTaskAttemptsPerExecutor")
+      .intConf
+      .createWithDefault(1)
```

Comment:
Nit: I worry about mixing mechanism (defining the config attribute) and policy (setting a default value) here. Granted, this change follows the existing pattern.